configs/config_mbedtls.h

```diff
@@ -0,0 +1,166 @@
+//
+// mbedTLS config
+//
+
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_CRYPTO_CONFIG__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_CRYPTO_CONFIG__
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Minimal Suite-B configuration
+ *
+ * Inherited from https://github.com/ARMmbed/mbedtls/blob/1e14827bebc8480786498f007304bd852ef953c6/configs/config-suite-b.h.
+ * Some of the configurations are disabled due to incompatibility with what we need. The configs that are disabled are still
+ * maintained here for documentation purpose only. Enabling them will break some parts of the SDK. Each disabled config has
+ * a "note" that explains why it needs to be disabled.
+ *
+ */
+
+/* System support */
+#define MBEDTLS_HAVE_ASM
+#define MBEDTLS_HAVE_TIME
+
+/* mbed TLS feature support */
+#define MBEDTLS_ECP_DP_SECP256R1_ENABLED
+#define MBEDTLS_ECP_DP_SECP384R1_ENABLED
+#define MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED
+#define MBEDTLS_SSL_PROTO_TLS1_2
+
+/* mbed TLS modules */
+#define MBEDTLS_AES_C
+#define MBEDTLS_ASN1_PARSE_C
+#define MBEDTLS_ASN1_WRITE_C
+#define MBEDTLS_BIGNUM_C
+#define MBEDTLS_CIPHER_C
+#define MBEDTLS_CTR_DRBG_C
+#define MBEDTLS_ECDH_C
+#define MBEDTLS_ECDSA_C
+#define MBEDTLS_ECP_C
+#define MBEDTLS_ENTROPY_C
+#define MBEDTLS_GCM_C
+#define MBEDTLS_MD_C
+#define MBEDTLS_NET_C
+#define MBEDTLS_OID_C
+#define MBEDTLS_PK_C
+#define MBEDTLS_PK_PARSE_C
+#define MBEDTLS_SHA256_C
+#define MBEDTLS_SHA512_C
+#define MBEDTLS_SSL_CLI_C
+#define MBEDTLS_SSL_SRV_C
+#define MBEDTLS_SSL_TLS_C
+#define MBEDTLS_X509_CRT_PARSE_C
+#define MBEDTLS_X509_USE_C
+
+/* For test certificates */
+#define MBEDTLS_BASE64_C
+#define MBEDTLS_CERTS_C
+#define MBEDTLS_PEM_PARSE_C
+
+/* Save RAM at the expense of ROM */
+#define MBEDTLS_AES_ROM_TABLES
+
+/* Save RAM by adjusting to our exact needs
+ *
+ * Note: disabled because we want RSA as well, which is much bigger that ECP.
```

Comment:
Not sure I understood the comment. We are disabling what and why?

Reply:
The `MBEDTLS_ECP_MAX_BITS` and `MBEDTLS_MPI_MAX_SIZE` flags are set in the original suite-b config, https://github.com/ARMmbed/mbedtls/blob/1e14827bebc8480786498f007304bd852ef953c6/configs/config-suite-b.h. This is an optimization to reduce the big number max size, but we can't reduce the max size since we're using RSA which requires a much bigger amount of bits. So, we're essentially disabling these optimizations.